### DISCUSSION TITLE
Non-standard namespaces in INSPIRE

### DISCUSSION THREAD

**Discussion by CLinGBA** [INSPIRE-MIF/helpdesk-validator#985] [Q&A] [answered]
Hello, 

I have a short question regarding namespaces:

is it legal to use additional namespaces when providing 2 datasets for one featureType?
e.g. ge1: and ge2: if providing different resolutions?

**Comment by KathiSchleidt** [comment]
From the XML/GML side of things, these alternative namespaces shouldn't be an issue
Not sure what the INSPIRE Validator does - have you tried submitting files with an alternative namespace (could just do a simple search/replace on an existing (valid) file to check)
:)

**Comment by sMorrone** [comment]
Dear @CLinGBA,
it seems that the validator raises an error, as expecting "ge" prefix  - see [test description](https://github.com/inspire-eu-validation/data-ge/tree/3.0/ge-core-as).
However, I would be interested in understanding the reason why you need two different names for same namespace ... Do you have publication issues with specific tool? your question reminds me of the GeoServer issue in providing more datasets for the same schema (issue solved thanks to the creation of "isolated workspaces") ... ?

**Comment by CLinGBA** [comment]
Dear @sMorrone,

we need different namespaces, because we want to provide the the same feature type in different scales and if we wanted to do this via GeoServer (& WFS) we have to use isolated workspaces. 

And if we do that we get as an output <ge1:GeologicUnit ...
which raises an error on the INSPIRE-Validator. (if I remember correctly)

**Comment by sMorrone** [comment]
Dear @CLinGBA , I am moving this issue to the Validator helpdesk for more specific support.

**Comment by fabiovinci** [comment]
Dear @CLinGBA,
could you please provide an example in order to reproduce the error? Thanks

**Comment by CLinGBA** [comment]
I'm sorry, right now we don't have any example. We tested this 2021 when we opened the issue. Since the the validator did not approve the output, we switched to a different solution, without a WFS

But for High-value datasets it is probably needed again so it would be good if the validator approves non-standard namespaces/prefixes like ge1:XXX

**Comment by fabiovinci** [comment, marked as answer]
Dear @CLinGBA,

I confirm that the Validator does not raise errors if non-standard namespaces/prefixes are used.

In the following zip file you can find an example dataset and its validation report:
[ge-core_sample_additional_namespace.zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/12049892/ge-core_sample_additional_namespace.zip)
